user_guide/content/modules/working_with_an_existing_work_item.adoc

```diff
@@ -1,90 +1,68 @@
 [#working_with_an_existing_work_item]
 = Working with an Existing Work Item
-//redo this, stopped here
-Use the Planner in the *Plan* tab to track your Work Items and Iterations using either the list view or the board view. The *Plan* tab is split into two panes:
 
-* The left pane displays the *Backlog* and *Board* views of Iterations and Work Items. For more information see <<about_iterations,About Iterations>>.
-* The right pane displays the Work Items within the selected Iteration or the backlog.
+Click btn:[Plan] at the top of the page to view the Planner.
 
-You can plan and track your Work Items in the right pane using either the hierarchical or flat list view of Work Items on the menu:Plan[Backlog] subtab or the Kanban-based board view on the menu:Plan[Board] subtab.
+The *Backlog* view is the default view in the Planner and it displays all the Iterations included in the space. Iterations are sorted into one of the following categories:
 
```

Comment:
Happy to have a discussion. Shall we wait for the meeting so we all get to be involved? :)